src/Parser/Warrior/Fury/Modules/BuffDebuff/EnrageUptime.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import StatisticBox from 'Main/StatisticBox';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class enrageUptime extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  suggestions(when) {
+    const enrageUptime = this.combatants.selected.getBuffUptime(SPELLS.ENRAGE.id) / this.owner.fightDuration;
+
+    when(enrageUptime).isLessThan(0.70)
```

Comment:
In the future the preferred way to create suggestions is to write them like this:

https://github.com/WoWAnalyzer/WoWAnalyzer/blob/e801ad76782be9ce19f5f55774b1a46e549f9107/src/Parser/Core/Modules/Items/Velens.js#L75-L85

This will make them easier to modify and use in the checklist.

For now you can leave it as is.